src/core/Button/styles.css

```diff
@@ -22,11 +24,16 @@
    icon get's pointer-events: none */
 .button i {
     pointer-events: none;
-    margin-right: 5px;
-    margin-bottom: 2px;
+    margin-right: 8px;
+    margin-left: -4px;
     vertical-align: middle;
 }
 
+.button i:only-child {
+    margin-right: 0;
+    margin-left: 0;
+}
+
 /* STATES */
```

Comment:
comments like these can be removed, unless they represent an odd behaviour not understandable from code